_data/meltano/extractors/tap-smoke-test/meltano.yml

```diff
@@ -0,0 +1,67 @@
+capabilities:
+- about
+- catalog
+- discover
+- schema-flattening
+- state
+- stream-maps
+description: Generates sample data to be used for testing.
+domain_url: https://github.com/meltano/tap-smoke-test
+executable: tap-smoke-test
+keywords:
+- meltano_sdk
+label: Smoke Test
+logo_url: /assets/logos/extractors/smoke-test.png
```

Comment:
What do you think about something like this? https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.iconfinder.com%2Ficons%2F728631%2Fexperiment_explosion_lab_laboratory_science_smoke_test_tube_icon&psig=AOvVaw38fYqDb-EjMOzIMwdRQ_Fu&ust=1681505418980000&source=images&cd=vfe&ved=0CBAQjRxqFwoTCPDPpvvdp_4CFQAAAAAdAAAAABAE

I like that better than a cloud

Reply:
@tayloramurphy yeah that one is good with me!